packages/discovery-react-components/package.json

```diff
@@ -29,7 +29,7 @@
     "@carbon/icons": ">= 10.5.0 < 11",
     "@ibm-watson/discovery-styles": "1.x.x",
     "carbon-components": ">= 10.6.0 < 11",
-    "carbon-components-react": ">= 7.6.0 < 8",
+    "carbon-components-react": ">= 7.7.0 < 8",
```

Comment:
If this is just to avoid some warnings when running tests do we really want to update the peer dependency too? Can't we just update the lock file?

Reply:
I agree, we don't necessarily need 7.7 to run, but locally to get rid of errors we can bump.

Peer dependencies in general should be lenient https://nodejs.org/es/blog/npm/peer-dependencies/